neptune/experiments.py

```diff
@@ -547,6 +560,123 @@ def download_artifact(self, filename, destination_dir):
 
         self._client.download_data(self._project, path, destination_path)
 
+    def download_sources(self, path=None, destination_dir=None):
+        """Download a directory or single file from experiment sources.
+
+        Download a subdirectory (or file) ``path`` from the experiment sources and save it in ``destination_dir``
+        as a ZIP archive.
+
+        Args:
+            path (:obj:`str`):
+                Path of a directory or file in experiment sources to be downloaded.
+                If ``None`` is passed, all source files will be downloaded.
+
+            destination_dir (:obj:`str`): The directory where the archive will be downloaded.
+                If ``None`` is passed, the archive will be downloaded to current working directory.
+
+        Raises:
+            `NotADirectory`: When ``destination_dir`` is not a directory.
+            `FileNotFound`: If a path in experiment sources does not exist.
```

Comment:
Is this raised in `download_artifact` as well? If so, please add it to the method's doc.